### DISCUSSION TITLE
Tech Blogの自分の記事だけを登録したい

### DISCUSSION THREAD

**Discussion by kbkn3** [lapras-inc/lapras-user-community#19] [Ideas]
はてなで運用しているTech Blogなど、複数人で執筆しているがサービスのアカウント管理上1つのメディアとして捉えられるタイプのブログの1記事だけスクレイピング対象にしようとすると全記事が登録されてしまいます。

1記事ずつ登録できるようにして頂けませんか？

**Comment by UserSuccessTeamLAPRAS** [comment]
LAPRASユーザーサポートです。
ご意見ありがとうございます！改善のご要望として開発チームに共有させていただきました。

今後ともLAPRASをよろしくお願いします。

**Reply by kbkn3** [reply]
ありがとうございます！

**Reply by rockymanobi** [reply]
@kbkn3 
LAPRASのSWEのrockyです。

ワークアラウンド的な対処方法で恐縮ですが、クロール対象サイトに追加頂く代わりに、`Selections` をご利用頂くことで、ブログサイトの特定の記事のみをポートフォリオに表示させることが可能です。

少し前に `Selections`に複数のURLを登録することができるようになりましたので、ご案内させていただきました 🙇 
よろしければお試しください！

**Reply by kbkn3** [reply]
@rockymanobi 
ありがとうございます！

試してみましたがSlectionsでも表示させて置けるのが嬉しいです。
